debezium-core/src/test/java/io/debezium/pipeline/notification/IncrementalSnapshotNotificationServiceTest.java

```diff
@@ -131,7 +131,7 @@ public void notifyTableScanCompleted() {
         Notification expectedNotification = new Notification("12345", "Incremental Snapshot", "TABLE_SCAN_COMPLETED", Map.of(
                 "connector_name", "connector-test",
                 "data_collections", "db.inventory.product,db.inventory.customer",
-                "scanned_collection", "db.inventory.product",
+                "scanned_collection", "\"db\".\"inventory\".\"product\"",
```

Comment:
WDYT to maintain the same format as for 'data_collections" field? Maybe here quoting is not strictly required.

Reply:
I did have the same thought. Ideally, debezium would use a double quoted identifier across all notifications, metrics etc. This removes ambiguity for the (rare) case that an identifier part includes a `.`

That could maybe be a different ticket if others agreed.

Im happy to go with a regular identifier for now. The main purpose of this ticket was to use the whole identifier for all table scan notifications.

Reply:
This is a good point. What is not clear to me is that you have reported 
>You cannot infer which table the notification is referring to when you have 2 tables with the same name in different schemas e.g. sch1.tbl and sch2.tbl. Both will produce a table scan completed notification with the 
"scanned_collection" of "tbl".

But the current implementation with the 'identifier' method already returns the full table name. Isn't it? Also the test states this. 

So is the problem just the quoting? Can you please clarify?

Reply:
I need to update the ticket. Currently, an incremental snapshot uses the complete table identifier but for an initial snapshot only the table name is given.

At the time of ticket creation, I assumed this was a design decision but it appears to be a bug.

This PR would fix that initial snapshot bug, then add double quotes to remove some ambiguity to both types of snapshot.

Reply:
Thinking about that, should we use double quotes? This seems to be database dependent as some of the database use double qutes, then there is for example SQL server that uses `[schema].[table]` format at db level.
We have `io.debezium.jdbc.JdbcConnection.quotedTableIdString(TableId)` that handles this nuance. Maybe we should introduce a spearate per connector class that would be repsonsible for such differences (frequently taking them ofo JdbcConnection) but also get rid of `useCatalogBeforeSchema` `io.debezium.relational.TableId` etc. WDYT?

Reply:
My thoughts are that we can use double quotes for all db's here. In my eyes this is more about wrapping the parts to remove ambiguity, than trying to keep inline with the db level syntax. 

We could use a different character to wrap the parts but I feel like this should be independent of the db.

Again, if we want to remove the ambiguity from all notifications, metrics etc. Im happy for that to be its own ticket. The key issues in this PR is resolving the initial snapshot bug, ensuring we use a full identifier instead of table name.

Reply:
I would propose to just solve the real bug with this PR and maybe open a further enhancement about the quoting to better discuss it. For example another solution can be to have a more structured data ( JSON Object) with `database`, `schema`, `table`.  

In that case you can change this issue type from `Feature Request` to `Bug`.

Reply:
Ill do that @mfvitale.

Reply:
https://issues.redhat.com/browse/DBZ-7350